src/Command.ino

```diff
@@ -198,6 +198,7 @@ String doExecuteCommand(const char *cmd, struct EventStruct *event, const char *
     }
     case 's': {
       COMMAND_CASE(       "save", Command_Settings_Save,   0); // Settings.h
+	  COMMAND_CASE(  "subscribe", Command_MQTT_Subscribe,  1);  // MQTT.h  //MFD: adding subscrie command
```

Comment:
Please use uncrustify to make sure the formatting is consistent.

Reply:
what settings are you using ?

Reply:
https://github.com/letscontrolit/ESPEasy/blob/mega/uncrustify.cfg

Reply:
where should I put this file? in the root of the project ?
I use VSCode.

Reply:
I think should be good now. please check

Reply:
but still no #ifdefs... #endif

Reply:
did not figured it, thanks :)

Reply:
> where should I put this file? in the root of the project ?
> I use VSCode.

It is already in the right place for Uncrustify.
Just install the extension in VS code and you're good to go :)

Reply:
done.